docs/release-notes/Percona-Everest-1.2.0-(2024-09-09).md

```diff
@@ -0,0 +1,204 @@
+
+
+## New features
+
+- [EVEREST-1103](https://perconadev.atlassian.net/browse/EVEREST-1103): Starting with Percona Everest 1.2.0, we've restricted actions based on RBAC roles, ensuring that users are explicitly granted access to the resources required for their specific roles. This enhances security and simplifies access control processes.
+
+- [EVEREST-1142](https://perconadev.atlassian.net/browse/EVEREST-1142): We have now added a [new command](../administer/rbac.md#validate-your-rbac-policy) for validating your RBAC policy to ensure that your RBAC policies are working as expected.
+
+- [EVEREST-1240](https://perconadev.atlassian.net/browse/EVEREST-1240): We have added support for PostgreSQL operator version 2.4.1. 
+
+- [EVEREST-1298](https://perconadev.atlassian.net/browse/EVEREST-1298): We have added support for MySQL operator version 1.15.0.
+
+- [EVEREST-1035](https://perconadev.atlassian.net/browse/EVEREST-1035): We've now included **Retention copies** for PostgreSQL as well when setting up backup schedules.
+
+
+## Improvements
+
+- [EVEREST-1165](https://perconadev.atlassian.net/browse/EVEREST-1165)- Due to limitations with the [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/), it is now required to upgrade all database operators concurrently with their components across any namespace.
+
+- [EVEREST-1212](https://perconadev.atlassian.net/browse/EVEREST-1212) - Starting with Percona Everest 1.2.0, you can now directly edit the monitoring endpoint from the database overview page, instead of having to use the Edit database wizard.
+
+
+- [EVEREST-1230](https://perconadev.atlassian.net/browse/EVEREST-1230): We've updated the **Resources** panel on the **Database overview** page to be independent instead of part of the **DB Details** panel and improved the overall look and feel of this page.
+
+
+- [EVEREST-1369](https://perconadev.atlassian.net/browse/EVEREST-1369): We've updated the documentation for the section on turning off telemetry.
+
+
+## The latest in APIs: What’s new and what’s deprecated
+
+
+=== "Newly added API endpoints"
+
+    Check out the new API endpoints we've added in Percona Everest 1.2.0:
+
+    |**No**|**API endpoints**|**Method**|
+    |---------|---------------------|---------|
+    |**1.**|`/namespaces/{namespace}/monitoring-instances`|a.`GET`</br> b.`POST`|
+    |**2.**|`/namespaces/{namespace}/monitoring-instances/{name}`|a.`GET`</br>b. `PATCH`</br>c.`DELETE`|
+    |**3.**|`/namespaces/{namespace}/backup-storages`|a.`GET`</br> b. `POST`|
+    |**4.**|`/namespaces/{namespace}/backup-storages/{name}`|a.`GET`</br> b.`POST`|
+    |**5.**|``/permissions``|a.`GET`|
+
+
+
+=== "Deprecated API endpoints"
+
+    This is the list of the API endpoints deprecated from Percona Everest:
+
+    |**No**|**API endpoints**|**Method**|
+    |---------|---------------------|---------|
+    |**1.** |**Endpoints deprecated from Percona Everest v1.1.0:**|
```

Comment:
We already discussed this offline but let's explicitly say that these endpoint that were deprecated in 1.1.0 have now been removed from 1.2.0